templates/tfengine/components/project/project/main.tf

```diff
@@ -1,28 +1,47 @@
-# Copyright 2020 Google Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+{{- /* Copyright 2020 Google LLC
```

Comment:
I think we have `Inc`. everywhere

Reply:
I guess you might get it from templates/tfengine/components/project/storage_buckets/main.tf, I guess we should change templates/tfengine/components/project/storage_buckets/main.tf to be Inc. to be consistent with everywhere else.

Reply:
It should actually be LLC now, not Inc. This changed in 2017.

Reply:
sounds good. then let's have a separate PR to change all the Inc. to LLC